lab-judy/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "lab-11-12-express-api",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab 11 single resource express api ======",
+  "main": "index.js",
+  "scripts": {
+    "start": "DEBUG='note*' node server.js",
+    "test": "DEBUG='note*' ./node_modules/mocha/bin/mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/JudyVue/lab-11-12-express-api.git"
+  },
+  "author": "Judy Vue",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/JudyVue/lab-11-12-express-api/issues"
+  },
+  "homepage": "https://github.com/JudyVue/lab-11-12-express-api#readme",
+  "devDependencies": {
+    "bluebird": "^3.4.6",
+    "body-parser": "^1.15.2",
+    "chai": "^3.5.0",
+    "debug": "^2.2.0",
+    "eslint": "^3.6.0",
+    "express": "^4.14.0",
+    "http-errors": "^1.5.0",
+    "mocha": "^3.0.2",
+    "morgan": "^1.7.0",
+    "node-uuid": "^1.4.7"
+  },
+  "dependencies": {
+    "mkdirp": "^0.5.1",
+    "superagent": "^2.3.0",
+    "bluebird": "^3.4.6",
+    "body-parser": "^1.15.2",
+    "chai": "^3.5.0",
+    "debug": "^2.2.0",
+    "eslint": "^3.6.0",
+    "express": "^4.14.0",
+    "http-errors": "^1.5.0",
+    "mocha": "^3.0.2",
+    "morgan": "^1.7.0",
+    "node-uuid": "^1.4.7"
+  }
+}
```

Comment:
You should manually go in and delete dependencies that are not in the right spot. devDependencies should include chai, eslint, and mocha. Everything else is a regular dependency, and shouldn't be duplicated in dev dependencies.